install/kubernetes/istio.yaml

```diff
@@ -885,10 +816,4 @@ spec:
       - name: istio-ca
         image: gcr.io/istio-testing/istio-ca:87901f278dd40e864b083409dfee9ca72e6a27e9
         imagePullPolicy: IfNotPresent
-        env:
-        - name: NAMESPACE
```

Comment:
did that version somehow get accidentally deployed to the shared test cluster ?
(trying to understand how come we have errors in the bad CA sha PR that appear to be bleeding on other PRs)

Reply:
No, it has not. Just checked the cluster, there is nothing there.